package.json

```diff
@@ -34,6 +34,9 @@
     "prettier": "npx prettier \"{src,test}/**/*.ts\" --check",
     "fix:prettier": "yarn prettier --write",
     "fix": "yarn fix:lint && yarn fix:prettier",
+    "test:build": "esr ./test/setup/build.ts",
+    "test:init": "DEBUG=amman* npx amman validator",
```

Comment:
Don't use `npx` in production or anywhere please as it is basically the same as doing `curl <some-url> | bash`
You already added `amman` as a `devDependency` (which also allows it to be cached for CI and avoids devs re-installing it over and over even if yarn ends up using local cache). 
Therefore you can remove `npx`.

Also you're depending on devs to have `npx` installed which is another problem ...